eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -776,24 +779,35 @@ object Router {
     */
   val ROUTE_MAX_LENGTH = 20
 
+  // The default amount of routes we'll search for when findRoute is called
+  val DEFAULT_ROUTES_AMOUNT = 3
 
   /**
-    * Find a route in the graph between localNodeId and targetNodeId, returns the route and its cost
+    * Find a route in the graph between localNodeId and targetNodeId, returns the route.
+    * Will perform a k-shortest path selection given the @param numRoutes and randomly select one of the result.
```

Comment:
The fact that we also add a ceiling on cost on top of the base algo deserves a mention here.

Reply:
Comment added